servicepulse/usage.md

```diff
@@ -89,6 +89,14 @@ An endpoint that is expected to no longer be used within the next 30 days. These
 
 Not an NServiceBus endpoint. These are broker queues or other messaging infrastructure that should not be included in NServiceBus licensing calculations.
 
+#### Gateway or Bridging Endpoint
+
+This is an infrastructure endpoint used by the Particular Platform<
```

Comment:
Should these `<` be `.`?

Reply:
Given that these have the same descriptions, will customers know which one to pick?